test/features/authentication_test.rb

```diff
@@ -0,0 +1,22 @@
+require './test/test_helper'
+
+class Authentication < ActionDispatch::IntegrationTest
+
+  def test_person_can_log_in_from_omniauth
+   person = Person.create(:provider => "github", :uid => "1", :first_name => "goldfish", :oauth_token => "token")
```

Comment:
Line is too long. [113/80]<br>Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 1) spaces for indentation.